qiita_pet/test/test_download.py

```diff
@@ -387,31 +387,31 @@ def tearDown(self):
 
     def test_download(self):
         # check failures
-        response = self.get('/download_public/')
-        self.assertEqual(response.code, 405)
-        self.assertEqual(response.reason, 'You need to specify both data (the '
-                         'data type you want to download - raw/biom) and '
-                         'study_id')
+        response = self.get('/public_download/')
+        self.assertEqual(response.code, 200)
```

Comment:
@antgonza I'm not sure about this change. Are these meant to be REST endpoints? If so, then they really do need to return the proper status code, as well as the appropriate message string.

Reply:
I'm actually not sure; my concern is that when a user goes to: https://qiita-rc.ucsd.edu/download_public/ in their browser this is what they see:
<img width="620" alt="Screen Shot 2019-07-09 at 12 07 49 PM" src="https://user-images.githubusercontent.com/2014559/60912263-39a27480-a242-11e9-97a2-1e823c6ae4f9.png"> but if they use wget:
```
$ wget --no-check-certificate "https://qiita-rc.ucsd.edu/download_public/"
--2019-07-09 12:08:52--  https://qiita-rc.ucsd.edu/download_public/
Resolving qiita-rc.ucsd.edu (qiita-rc.ucsd.edu)... 132.249.211.60
Connecting to qiita-rc.ucsd.edu (qiita-rc.ucsd.edu)|132.249.211.60|:443... connected.
WARNING: cannot verify qiita-rc.ucsd.edu's certificate, issued by ‘emailAddress=jdereus@ucsd.edu,CN=qiita-rc.ucsd.edu,O=UCSD,L=La Jollaq,ST=California,C=US’:
  Self-signed certificate encountered.
HTTP request sent, awaiting response... 405 You need to specify both data (the data type you want to download - raw/biom) and study_id
2019-07-09 12:08:52 ERROR 405: You need to specify both data (the data type you want to download - raw/biom) and study_id.
```

Thus, thought that changing to 200 so the user can see the error as this will minimize enquiries.